modules/explorer-core/src/main/scala/org/ergoplatform/explorer/db/repositories/DexOrdersRepo.scala

```diff
@@ -0,0 +1,114 @@
+package org.ergoplatform.explorer.db.repositories
+
+import cats.Monad
+import fs2.Stream
+import org.ergoplatform.explorer.Err.DexErr.{
+  DexBuyOrderAttributesFailed,
+  DexSellOrderAttributesFailed
+}
+import org.ergoplatform.explorer.Err.RequestProcessingErr.{
+  Base16DecodingFailed,
+  ErgoTreeDeserializationFailed
+}
+import org.ergoplatform.explorer.db.DexContracts
+import org.ergoplatform.explorer.db.algebra.LiftConnectionIO
+import org.ergoplatform.explorer.db.models.aggregates.{
+  DexBuyOrderOutput,
+  DexSellOrderOutput
+}
+import org.ergoplatform.explorer.{HexString, TokenId}
+import org.ergoplatform.explorer.syntax.stream._
+import tofu.Raise.ContravariantRaise
+
+/** [[ExtendedOutput]] for DEX sell/buy orders data access operations.
+  */
+trait DexOrdersRepo[D[_], S[_[_], _]] {
```

Comment:
Same rule for repos -- these methods are expected to be in `OutputRepo`.

Reply:
Done.